src/vs/editor/browser/widget/diffEditorWidget.ts

```diff
@@ -1571,6 +1588,10 @@ abstract class DiffEditorWidgetStyle extends Disposable {
 
 	public abstract setEnableSplitViewResizing(enableSplitViewResizing: boolean): void;
 	public abstract layout(): number;
+
+	setBoundarySashes(_sashes: IBoundarySashes): void {
```

Comment:
Should this be abstract?

Reply:
In that case I would still need a noop implementation in `DiffEditorWidgetInline`, so I chose otherwise.